src/XMakeTasks/Microsoft.Xaml.targets

```diff
@@ -13,7 +13,7 @@
         the .NET Framework directory and thus will no longer be the right answer. Override it to point
         to the correct .NET Framework location. --> 
    <PropertyGroup>
-      <XamlBuildTaskPath Condition="'$(XamlBuildTaskPath)' == ''">$(MSBuildFrameworkToolsPath)</XamlBuildTaskPath>
+      <XamlBuildTaskPath Condition="'$(XamlBuildTaskPath)' == ''">$(MSBuildToolsPath)</XamlBuildTaskPath>
```

Comment:
Can you set this as a default inside `Microsoft.Xaml.targets`, using `$(MSBuildThisFileDirectory)` or something? That seems more robust (for example, it'd work if you packaged the task assembly in a NuGet package, which is desirable at some point).